### PR TITLE
Add simple example that shows how to style the editable.

### PR DESCRIPTION
**Pull request by jenstroeger**
I’ve been playing around with the styling of editable elements, and maybe this one’s useful as an illustration. Maybe use a fixed-width font?

### CODE REVIEW

File: examples/index.js [hunk 1]
```diff
@@ -19,13 +19,23 @@ $(() => {
...
   editable.on('change', (elem) => {
     if (elem === secondExample) updateCode(elem)
   })
 
+  // Styling
+  // -------
+
+  $('select[name="editable-styles"]').on('change', function() {
+    $('.styling-example p').removeClass('example-style-default example-style-dark')
+                           .addClass('example-style-' + $(this).val())
```
Comment (jenstroeger) [inline examples/index.js:36]: I wanted to keep it explicit, hence the two CSS styles `default` and `dark`. We could simplify the code, remove the empty default styling, and call [`toggleClass()`](https://api.jquery.com/toggleClass/). Whichever works for you, @peyerluk.

[the remark this answers is not in the document]
Comment (jenstroeger) [reply examples/index.js]: Fixed https://github.com/livingdocsIO/editable.js/pull/195/commits/1e67ab7f1dce1dc1846278979f2f635eb094df92

File: examples/index.js [hunk 2]
```diff
@@ -19,13 +19,23 @@ $(() => {
...
   updateCode(secondExample)
 
   editable.on('change', (elem) => {
     if (elem === secondExample) updateCode(elem)
   })
 
+  // Styling
+  // -------
+
+  $('select[name="editable-styles"]').on('change', function() {
```
[the remark this answers is not in the document]
Comment (jenstroeger) [reply examples/index.js]: Fixed https://github.com/livingdocsIO/editable.js/pull/195/commits/1e67ab7f1dce1dc1846278979f2f635eb094df92

**Comment by peyerluk** [comment]
Maybe the styling example on the demo page misses a short description and a code example? 

I feel this styling example is a bit more how you can work with editable.js and less about what editable can do for you. The only thing editable.js does here is copying the class names on split and enter events. But with a sentence or two explaining what this styling example tries to teach I think this could be useful in understanding better how you're meant to work with editable.js.

**Comment by jenstroeger** [comment]
> I feel this styling example is a bit more how you can work with editable.js and less about what editable can do for you.

True.

> But with a sentence or two explaining […]

You mean like an explanatory paragraph or caption below the editable?

**Comment by jenstroeger** [comment]
@peyerluk, I added an explanation and squashed the commits into a single one.

**Comment by jenstroeger** [comment]
@peyerluk, I rebased on yesterday’s master if you’d like to merge. Looks very spiffy, methinks 😉

**Pull request merged**